tests/test_e2e_14_mef_eline.py

```diff
@@ -118,22 +118,6 @@ def test_005_create_evc_on_nni(self):
         assert data['enabled'] == True
         assert data['active'] == True
 
-        # Verify connectivity
-        h6, h1 = self.net.net.get('h6', 'h1')
-        h6.cmd('ip link add link %s name vlan100 type vlan id 100' % (h6.intfNames()[0]))
-        h6.cmd('ip link set up vlan100')
-        h6.cmd('ip addr add 10.1.0.6/24 dev vlan100')
-        h1.cmd('ip link add link %s name vlan100 type vlan id 100' % (h1.intfNames()[0]))
-        h1.cmd('ip link set up vlan100')
-        h1.cmd('ip addr add 10.1.0.1/24 dev vlan100')
-
-        result = h6.cmd('ping -c1 10.1.0.1')
-        assert ', 0% packet loss,' in result
-
-        # clean up
-        h6.cmd('ip link del vlan100')
-        h1.cmd('ip link del vlan100')
-
```

Comment:
This connectivity test would not work since the UNIs are NNIs and these do not connect to hosts. I was thinking adding assertions to test the quantity of flows in the topology. Is there an option send packet with a certain vlan between switches?

Reply:
@Alopalao good observation. I think a good compromise to assert this traceability would be to run a trace with the `sdntrace` NApp since that will also generate packets in/outs on wire, and then you assert the expected trace result. Wdyt? 

> Is there an option send packet with a certain vlan between switches?

Yes, there are also a way to inject and build any packet with `scapy` for instance, but that usually also entails to keep sniffing the expected receiving interface and filter and assert accordingly with external tools, so way more work than using anything else that's already available on mininet or any other trace that we have available on our NApps.

Reply:
`sdntrace` trace sounds fair, added.